harley/schema_helpers.py

```diff
@@ -0,0 +1,20 @@
+from typing import OrderedDict
+from polars import DataType
+
+def print_schema_as_code(schema: OrderedDict[str, DataType], prepend_pl: bool = False)-> str:
+    """
+    Print schema as valid code.
+    Optional `prepend_pl` flag for prepending 'pl.' to the column types.
+    """
+    if prepend_pl:
+        schema = OrderedDict(
+        [
+            (field, "pl."+ str(d_type))
+            for field, d_type in schema.items()
+        ]
+    )
+    out = "OrderedDict(["
```

Comment:
Could just use `str(schema)`? Or is this to ensure we have `"..."` for strings?

Reply:
Yeah a tiny thing, and probably not worth it but `ruff`/`black` will format strings to double quotes so may as well do it the "recommended" way.

Reply:
Possibly. But what about nested schemas? (e.g. with list / struct)
```python
data = [{"name":{"first_name":"Charles","last_name":"Spring"},"age":18,"nicknames":["Charlie","Char"]},{"name":{"first_name":"Nicholas","last_name":"Nelson"},"age":19,"nicknames":["Nick"]}]
df = pl.DataFrame(data)
print(df.schema)
# Schema([('name', Struct({'first_name': String, 'last_name': String})), ('age', Int64), ('nicknames', List(String))]
harley.print_schem_as_code(df.schema, prepend_pl=True)
# OrderedDict([("name", pl.Struct({'first_name': String, 'last_name': String})),("age", pl.Int64),("nicknames", pl.List(String)),])
```